pkg/adapterManager/manager.go

```diff
@@ -149,6 +159,21 @@ func (m *Manager) Execute(ctx context.Context, cfgs []*config.Combined, attrs at
 		}
 	}
 
+	bags := make([]*attribute.MutableBag, numCfgs)
+	for i, r := range results {
+		bags[i] = r.responseBag
+	}
+
+	err := responseBag.Merge(bags)
```

Comment:
You can combine these to `if err := responseBag.Merge(bags); err != nil { ... }`